portal/src/graphql/portal/LockoutSettings.tsx

```diff
@@ -72,71 +78,212 @@ function useNumberOnChange<T extends State>(
   );
 }
 
+function formatOptionalHour(hour: number): string {
```

Comment:
To format an optional number, we need to use two values.

```
{xIsPresent, select, true {{x, plural, one{# unit} other{# units}}} other{none}}
```

```
{
  xIsPresent: String(x != null) // a string of "true" or "false", which can be used in select
  x: x,
}
```

The problem is this function is that it returns `none` which is language specific.

Reply:
@louischan-oursky I am just trying to return something which the formatter will understand, it can be 0, none, null or anything other thing, as long as we handled it in the message.